docs/about/release-notes.rst

```diff
@@ -18,6 +18,7 @@ Features
 * Made it possible to use ``-1`` for one of the dimension sizes when using ``sc.fold`` `#2414 <https://github.com/scipp/scipp/pull/2414>`_.
 * Added ``quiet`` argument to :py:func:`scipp.transform_coords` `#2420 <https://github.com/scipp/scipp/pull/2420>`_.
 * Added support for strides for positional slicing. Negative strides are not supported for now `#2423 <https://github.com/scipp/scipp/pull/2423>`_.
+* ``sc.zeros_like``, ``sc.ones_like``, ``sc.empty_like``, and ``sc.full_like`` now accept and return data arrays `#2425 <https://github.com/scipp/scipp/pull/2425>`_.
```

Comment:
We have started to link to functions from the release notes, use `:py:func:`scipp.zeros_like`, etc.